### PR TITLE
Add A Place To Belong Fanart Contest Results newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
Big one, eyes welcome.

### CODE REVIEW

File: news/2022-05-08-aptb-fanart-results.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+[![](https://assets.ppy.sh/contests/142/banners/Sjao.jpg)](https://assets.ppy.sh/contests/142/winners/Sjao.png)
+
+[**Sjao**](https://osu.ppy.sh/users/7295733) returns once again with a windswept pippi and Mocha before a blissful springtime vista that **2646** voters could not help but adore.
+
+[![](https://assets.ppy.sh/contests/142/banners/ZunK.jpg)](https://assets.ppy.sh/contests/142/winners/ZunK.png)
+
+Springtime camping is on the cards in [**ZunK**](https://osu.ppy.sh/users/1070783)'s entry showcasing pippi, Mocha and the Dean "peppy" Herbert in a most huggable doll form. **2594** voters likely smashed the vote button so hard that they sprained something afterwards.
+
+Don't let the magnificence of the doll distract you - Mocha's rocking some serious style in this one. A don is chilling out in the back there too!
```
Comment (Walavouchey) [inline news/2022-05-08-aptb-fanart-results.md:55]:
```suggestion
Don't let the magnificence of the doll distract you — Mocha's rocking some serious style in this one. A don is chilling out in the back there too!
```

File: news/2022-05-08-aptb-fanart-results.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+Is it a date? A field trip for a school project? Are we the fish Mocha is pointing at? So many questions, but we'll thankfully have the next few months to think it over while this wonderful piece makes its well-deserved main menu appearance.
+
+[![](https://assets.ppy.sh/contests/142/banners/DGen.jpg)](https://assets.ppy.sh/contests/142/winners/DGen.png)
+
+[**DGen**](https://osu.ppy.sh/users/13290907) won **2813** votes with a rendition of pippi and Chirou enjoying a vast assortment of Japanese foods, complete with "Choked Top Play Soy Sauce (with Extra Salt)", presumably freshly harvested from your favourite osu! streamer or top player of choice.
+
+[![](https://assets.ppy.sh/contests/142/banners/PacifiedPaints.jpg)](https://assets.ppy.sh/contests/142/winners/PacifiedPaints.png)
+
+The very un-aptly named [**PacifiedPaints**](https://osu.ppy.sh/users/8377453) produced a punchy punk picture of pippi and Mocha engaging in some urban art to the approval of **2727** voters.
```
Comment (Walavouchey) [inline news/2022-05-08-aptb-fanart-results.md:43]: [it's a word](https://www.thefreedictionary.com/unaptly)

```suggestion
The very unaptly named [**PacifiedPaints**](https://osu.ppy.sh/users/8377453) produced a punchy punk picture of pippi and Mocha engaging in some urban art to the approval of **2727** voters.
```

**Review by peppy — APPROVED** [review]

File: news/2022-05-08-aptb-fanart-results.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+[![](https://assets.ppy.sh/contests/142/banners/Swarmsii.jpg)](https://assets.ppy.sh/contests/142/winners/Swarmsii.png)
+
+Hark! The fantastical imaginings of [**Swarmsii**](https://osu.ppy.sh/users/11716111) depicting an adventuring ensemble of Mocha, Yuzu and Chirou impressed **2213** denizens of the realm.
+
+Of course this fantasy-Yuzu is haggling for fruit, and of course the fantasy-Mocha is carrying a very sharp stick. Of course they are. It just makes sense!
+
+[![](https://assets.ppy.sh/contests/142/banners/Kwms024.jpg)](https://assets.ppy.sh/contests/142/winners/Kwms024.png)
+
+Dialling attitude up to maximum, [**Kwms024**](https://osu.ppy.sh/users/11516014) ruthlessly intimidated (or just straight up convinced, your choice) **2207** voters into supporting their urbane osu! piece.
```
Comment (Walavouchey) [inline news/2022-05-08-aptb-fanart-results.md:71]: image


```suggestion
Dialling attitude up to maximum, [**Kwms024**](https://osu.ppy.sh/users/11516014) ruthlessly intimidated (or just straight up convinced, your choice) **2207** voters into supporting their urban osu! piece.
```

File: news/2022-05-08-aptb-fanart-results.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+There is a *lot* going on in this piece, and the more you look at it, the more you see. Wonderfully stylized and beautifully executed, though unfortunately pippi's *slightly* provocative choice of attire means that it won't be actually making it to the main menu despite being a bonafide finalist. That's a bummer. The piece still owns, though.
```
Comment (Walavouchey) [inline news/2022-05-08-aptb-fanart-results.md:73]: looks like this article uses mostly british english

```suggestion
There is a *lot* going on in this piece, and the more you look at it, the more you see. Wonderfully stylised and beautifully executed, though unfortunately pippi's *slightly* provocative choice of attire means that it won't be actually making it to the main menu despite being a bonafide finalist. That's a bummer. The piece still owns, though.
```

File: news/2022-05-08-aptb-fanart-results.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+[![](https://assets.ppy.sh/contests/142/banners/Dem4eg-.jpg)](https://assets.ppy.sh/contests/142/winners/Dem4eg-.png)
+
+The cheers of **2137** voters lifted up [**Dem4eg-**](https://osu.ppy.sh/users/7206818) piece of pippi and Mocha taking center stage as charming idols, complete with a fanatic crowd cheering and swinging their pastel glowsticks.
```
Comment (Walavouchey) [inline news/2022-05-08-aptb-fanart-results.md:77]:
```suggestion
The cheers of **2137** voters lifted up [**Dem4eg-**](https://osu.ppy.sh/users/7206818)'s piece of pippi and Mocha taking centre stage as charming idols, complete with a fanatic crowd cheering and swinging their pastel glowsticks.
```

File: news/2022-05-08-aptb-fanart-results.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+[![](https://assets.ppy.sh/contests/142/banners/lILY1231.jpg)](https://assets.ppy.sh/contests/142/winners/lILY1231.png)
+
+[**lILY1231**](https://osu.ppy.sh/users/29265711)'s rendition of pippi and Alisa enjoying some sparklers ignited the hearts of **2097** voters. How can you say no to those faces?
+
+[![](https://assets.ppy.sh/contests/142/banners/tehfire.jpg)](https://assets.ppy.sh/contests/142/winners/tehfire.png)
+
+[**tehfire**](https://osu.ppy.sh/users/7082924)'s most orbital arrangement of basically the entire official cast (plus the Taikonator) floating in space impressed **2055** voters. If you look really closely, they're all there, and what could be mistaken for solar panels is in fact, a osu!mania playfield. Or could be.
```
Comment (Walavouchey) [inline news/2022-05-08-aptb-fanart-results.md:85]:
```suggestion
[**tehfire**](https://osu.ppy.sh/users/7082924)'s most orbital arrangement of basically the entire official cast (plus the Taikonator) floating in space impressed **2055** voters. If you look really closely, they're all there, and what could be mistaken for solar panels is in fact, an osu!mania playfield. Or could be.
```

File: news/2022-05-08-aptb-fanart-results.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+What's that? You're worried about Yuzu being cut loose? Don't worry, he'll be just fine.
+
+Probably.
+
+[![](https://assets.ppy.sh/contests/142/banners/Porukana.jpg)](https://assets.ppy.sh/contests/142/winners/Porukana.png)
+
+**2003** voters gazed across the neon lights of a cyberpunky cityscape with [**Porukana**](https://osu.ppy.sh/users/12992775)'s rendition of pippi and Tama in special osu! branded athletics wear. Looking stylish!
```
Comment (Walavouchey) [inline news/2022-05-08-aptb-fanart-results.md:93]:
```suggestion
**2003** voters gazed across the neon lights of a cyberpunky cityscape with [**Porukana**](https://osu.ppy.sh/users/12992775)'s rendition of pippi and Tama in special osu!-branded athletic wear. Looking stylish!
```

File: news/2022-05-08-aptb-fanart-results.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+[![](https://assets.ppy.sh/contests/142/banners/araran.jpg)](https://assets.ppy.sh/contests/142/winners/araran.png)
+
+**1886** voters couldn't help but fall for the charms of [**araran**](https://osu.ppy.sh/users/8937198)'s fox-type pippi, Chirou and an eared don enjoying some time at their local slider gallery. There's a joke to be made here about tech maps, but I will not be the one to do it.
+
+---
+
+[*Download an archive containing high-quality images for all the finalist entries here!*](https://assets.ppy.sh/contests/142/APTB2022-Finalists.zip)
```
Comment (Walavouchey) [inline news/2022-05-08-aptb-fanart-results.md:101]:
```suggestion
[Download an archive containing high-quality images for all the finalist entries here!](https://assets.ppy.sh/contests/142/APTB2022-Finalists.zip)
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**